gateway/js/app.js

```diff
@@ -888,6 +891,9 @@ function myAccount()
 
 function logged()
 {
+	if (typeof(mylocalStorage['privKeyInfoAck'] == 'undefined')) {
+		mylocalStorage['privKeyInfoAck'] = 0;
```

Comment:
We need to investigate on HTTP cookie to handle this. That we can cover in next PR.

Reply:
ack. session management overall will be overhauled in future.